circle.yml

```diff
@@ -0,0 +1,15 @@
+machine:
```

Comment:
I'd say there's no clear winner, CircleCI is around for a while and used widely (incl. Angular project). Also see popularity in terms of search: https://trends.google.co.uk/trends/explore?q=travisci,circleci

You OK to leave it as it is?